articles/other/other-ref-sla-definition.md

```diff
@@ -29,239 +29,262 @@ Subject to the Service Level Agreement (SLA) limitations detailed below, the ser
+**Key exclusions**           | The following are examples of what is not covered by the SLA:<ul><li>Faults within your control, such as client application issues<li>Faults within external connectivity providers (for example DDoS-protected internet, PSN, Janet or HSCN) and components co-located at UKCloud</ul>
 
 ### Disaster Recovery as a Service (DRaaS)
 
 Although we don't offer an SLA relating the performance of the Zerto software, we do provide one relating to the availability of the UKCloud for VMware platform that hosts the solution.
 
-&nbsp; | &nbsp;
--------|---------------------------
-&nbsp; | **UKCloud for VMware (POWER)**
-**Service level agreement** | 99.99%
-**Portal level agreement**  | 99.90%
+&nbsp;                       | UKCloud for VMware (POWER)
+-----------------------------|---------------------------
+**Service level agreement**  | 99.99%
 **Availability calculation** | Availability is calculated based on the number of hours in the billing month (for example, 744 hours for months with 31 days), excluding any emergency maintenance.
-**Planned maintenance** | Included in calculations
-**Measurement of SLA** | Unavailability applies to existing storage that has become unresponsive due to a fault recognised at the IaaS layer or below, that is, the fault is within UKCloud-controlled components, such as the physical host availability, storage, power and internal networking such as physical firewalls and routers. Although we don't offer an SLA relating to the performance of the Zerto software, we do offer an SLA around the availability of the self-service portal, and one relating to the availability of the UKCloud for VMware platform that hosts the solution.
-**Key exclusions** | The following are examples of what is not covered by the SLA: <br>- Faults within your control, such as client application issues<br>- Faults within external connectivity providers (for example DDoS- protected internet, PSN, Janet or HSCN) and components colocated at UKCloud<br>- Replication faults introduced by your environment, or your implementation of Zerto
-**Service Credit** | 5% of monthly spend per 5% below service level target or part thereof for affected compute platform. <br> 1% of monthly spend per 1% below service level target or part thereof for the UKCloud API and Portal.
+**Planned maintenance**      | Included
+**Measurement of SLA**       | Unavailability applies to existing storage that has become unresponsive due to a fault recognised at the IaaS layer or below, that is, the fault is within UKCloud-controlled components, such as the physical host availability, storage, power and internal networking such as physical firewalls and routers. Although we don't offer an SLA relating to the performance of the Zerto software, we do offer an SLA around the availability of the self-service portal, and one relating to the availability of the UKCloud for VMware platform that hosts the solution.
```

Comment:
looking at SD - Measurement of SLA is as follows... Unavailability applies to existing VMs that become unresponsive due to a fault recognised at the IaaS layer or below, that is, the fault is within UKCloud-controlled components, such as the physical host availability, storage, power and internal networking such as physical firewalls and routers.

Reply:
This section is for DRaaS (rather than the VMware service), so the reference to storage applies in this case